damus/Views/Chat/ChatroomThreadView.swift

```diff
@@ -130,6 +210,15 @@ struct ChatroomThreadView: View {
                     user_just_posted_flag = false
                 }
             }
+            .onChange(of: self.untrusted_network_expanded) { newValue in
+                guard newValue, untrusted_count > 0 else {
+                    return
+                }
+
+                // Give 0.5 seconds for disclosure group to expand before scrolling.
+                // Otherwise, there will be a race condition and won't render properly until the next redraw.
+                scroll_to_event(scroller: scroller, id: ChatroomThreadView.untrusted_network_scroll_id, delay: 0.5, animate: true, anchor: .top)
```

Comment:
Oh, actually 0.5s is pretty long and makes the UX sluggish. I'll make it shorter.

Reply:
Changed to 0.2 and seems to work, at least on my device.